syntax = "proto3";

message PysisCommand {
  string command = 1;
  repeated string from_ = 2;
  map<string, string> args = 3;
}

message PysisResult {
  int32 return_code = 1;
  repeated string to = 2;
  string stderr = 3;
}

service Pysis {
  rpc Isis(PysisCommand) returns (PysisResult) {}
}